### DISCUSSION TITLE
Allow external access to a specific route during development

### DISCUSSION THREAD

**Discussion by jisol** [quasarframework/quasar#14950] [General - Components / Directives / etc] [answered]
Example: the user types in the address bar of his browser the URL **http://my_app.dv:9000/post/140** (URL that was shared with the user, for example, by email).
**During development**, how to enable the correct loading of this route, avoiding a redirect to the home page?
Dev mode: spa; 
Pkg quasar: v2.10.1; 
Pkg @quasar/app-vite: v1.1.3;
vue-router: v4.1.6

**Comment by MilosPaunovic** [comment, marked as answer]
Amend router guards, if you have 'em. But, this is more of a `vue-router` stuff, not really related to Quasar, so I would suggest to ask that question on their repo.
